### ISSUE TITLE
config.AddOcelotWithSwaggerSupport reloadOnChange files

### ISSUE THREAD

**Issue by naji-makhoul** [Burgyn/MMLib.SwaggerForOcelot#234]
reloadOnChange file when use AddOcelotWithSwaggerSupport

```
config.AddOcelotWithSwaggerSupport((o) => {
                  o.Folder = "OcelotConfiguration";                   
            }).AddEnvironmentVariables();

```

like when add json file : 
  ```
config.AddJsonFile($"ocelot.{hostingContext.HostingEnvironment.EnvironmentName}.json", optional: false, reloadOnChange: true)
                       .AddEnvironmentVariables();
```

**Comment by Burgyn** [comment]
Hi,
thank you for your issue.

Unfortunately, you didn't write much. I cannot judge whether it is a question, a comment, a request, ...

Could you please be more specific? Try to describe current behavior, expected behavior, ... Ideally, add an example.

**Comment by naji-makhoul** [comment]
Hi . 
i need to reload ocelot file when the file is changes on runtime without restart the app 

when i set the ocelot config  files with  AddOcelotWithSwaggerSupport and change the routes in the file its not updated on runtime

when i use AddJsonFile with reloadOnChange = true , its updated

**Comment by Burgyn** [comment]
Hi,

which version do you use? This was fix in version `5.2.0` (PR #230)

**Comment by naji-makhoul** [comment]
I use version 4.9.0 , I will update and check it again, thank you
